release-tools/src/test/java/org/springframework/data/release/jira/TicketsUnitTests.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.data.release.jira;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.*;
+import static org.springframework.test.util.AssertionErrors.fail;
+
+import java.util.Collections;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.springframework.data.release.model.Iteration;
+import org.springframework.data.release.model.ReleaseTrains;
+
+/**
+ * Unit tests for {@link Tickets}.
+ * 
+ * @author Mark Paluch
+ */
+public class TicketsUnitTests {
+
+	@Rule public final ExpectedException expectedException = ExpectedException.none();
```

Comment:
If you don't check anything but the exception being thrown at all, simply using the `@Test(expected=…)` is fine.